plugin/healthchecker/heartbeatmemory/checker_memory_test.go

```diff
@@ -0,0 +1,36 @@
+package heartbeatmemory
```

Comment:
hello, pls add the copyright headers to this source code to make workflow passed

Reply:
added